cdc/processor/processor.go

```diff
@@ -776,11 +762,9 @@ func (p *processor) doGCSchemaStorage() {
 		// schemaStorage is nil only in test
 		return
 	}
-	// Delay GC to accommodate pullers starting from a startTs that's too small
-	// TODO fix startTs problem and remove GC delay, or use other mechanism that prevents the problem deterministically
-	gcTime := oracle.GetTimeFromTS(p.changefeed.Status.CheckpointTs).Add(-schemaStorageGCLag)
-	gcTs := oracle.ComposeTS(gcTime.Unix(), 0)
-	p.schemaStorage.DoGC(gcTs)
```

Comment:
What is the `20min` GC delay used for originally?

Reply:
It was used to accommodate some buggy behaviors in the old processor, where a puller is started from a timestamp less than the global checkpoint. The new owner does not have this problem.